sdcflows/interfaces/utils.py

```diff
@@ -384,3 +404,40 @@ def _reoblique(in_epi, in_plumb, in_field, in_mask=None, newpath=None):
         masknii.__class__(masknii.dataobj, epinii.affine, hdr).to_filename(out_files[2])
 
     return out_files
+
+
+def _pad_num_slices(in_file, newpath=None):
+    """
+    Check if image has an even number of slices.
```

Comment:
```suggestion
    Ensure the image has even number of slices to avert TOPUP's segfault.
    
    Check if image has an even number of slices.
```

There should be an empty line between the oneliner description of the function and the long description.